nttt/utilities.py

```diff
@@ -1,14 +1,15 @@
 from shutil import copy2, copystat, Error, ignore_patterns
 import os
 import os.path
-import codecs
+import eol
 
 def get_file(file_name):
-    with codecs.open(file_name, encoding='utf-8') as f:
-        return f.read()
+    with open(file_name, encoding='utf-8') as f:
```

Comment:
Using builtin `open` instead of `codecs.open` because the former one supports "universal newline mode" out of the box: https://softwareengineering.stackexchange.com/questions/298677/why-is-universal-newlines-mode-deprecated-in-python

This means that when text content is read for a file, line ending are always represented by `\n`